extensions/wikia/Wall/WallHistoryController.class.php

```diff
@@ -217,8 +219,18 @@ private function getFormatedHistoryData( $history, $threadId = 0 ) {
 				$history[$key]['msgurl'] = $messagePageUrl;
 
 				$msgUser = $wm->getUser();
+				$msgPage = Title::newFromText( $msgUser->getName(), $ns );
+				if ( empty( $msgPage ) ) {
```

Comment:
I don't think this is the right place to make changes like this.  ```ArticleComment``` is the code that works with ```Revision``` objects which implement this feature.  It manipulates the ```User``` object it loads by setting the ```User```'s ```mName``` field directly (according to Tor) to preserve the original author of the thread:
https://github.com/Wikia/app/blob/d54b8898c9b02f32ca9aa5a292567ae9202369fc/extensions/wikia/ArticleComments/classes/ArticleComment.class.php#L217-L218

Now that we understand this better I'm going to remove the logging, add defaults and leave a comment behind.

I'm going to close out this ticket.  We're in the middle of replacing this system and putting more effort into an edge case no user has reported is not a good use of our resources that are already stretched.  The return on investment in this ticket is extremely low and is only getting smaller.